### PR TITLE
ENG-2330: Add basic code block component

### PR DESCRIPTION
**Pull request by ksekhri**

### CODE REVIEW

File: src/components/Kv/KvCodeBlock.vue [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<template>
+	<div class="code" :class="`${nowrap ? 'nowrap' : ''}`">
+		<pre v-if="code">{{ code }}</pre>
+		<slot v-else></slot>
```
Comment (mcstover) [inline src/components/Kv/KvCodeBlock.vue]: Please wrap whatever is rendered in a <code> element.

File: src/components/Kv/KvCodeBlock.vue [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+			default: false,
+		},
+	},
+};
+</script>
+
+<style lang="scss" scoped>
+@import 'settings';
+
+.code {
```
Comment (mcstover) [inline src/components/Kv/KvCodeBlock.vue]: please move these to typography.scss globals

File: src/components/Kv/KvMultiCodeBlock.vue [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+<script>
+import KvCodeBlock from '@/components/Kv/KvCodeBlock';
+
+export default {
+	components: {
+		KvCodeBlock,
+	},
+	data() {
+		return {
+			code: [
```
Comment (mcstover) [inline src/components/Kv/KvMultiCodeBlock.vue]: Can we move this out into the parent that "uses" the component?

**Comment by ksekhri** [comment]
@mcstover @go-bears just pushed a version which fulfills all ticket requirements :)

File: src/components/Kv/KvCodeBlock.vue (hunk 1 above)
Comment (ksekhri) [reply src/components/Kv/KvCodeBlock.vue]: Cool, done!

File: src/components/Kv/KvCodeBlock.vue (hunk 2 above)
Comment (ksekhri) [reply src/components/Kv/KvCodeBlock.vue]: Cool, done 👍

File: src/components/Kv/KvMultiCodeBlock.vue (hunk 3 above)
Comment (ksekhri) [reply src/components/Kv/KvMultiCodeBlock.vue]: Done :+1:

**Pull request merged**